### ISSUE TITLE
[doc] missing `import` documentation

### ISSUE THREAD

**Issue by Maxwell2022** [newrelic/terraform-provider-newrelic#2574]
Hi there,

### Terraform Version

Latest - https://registry.terraform.io/providers/newrelic/newrelic/latest/docs

### Affected Resource(s)

- `newrelic_notification_destination`
- `newrelic_notification_channel`

### Terraform Configuration

N/A

### Actual Behavior

missing documentation section on importing resources. Under Slack example there is a terraform import command:
```hcl
terraform import newrelic_notification_destination.foo <destination_id>
```

There is no information on how to find this `<destination_id>`. 

Typically imports have their own section in the resource documentation page, at the very end.

<img width="180" alt="image" src="https://github.com/newrelic/terraform-provider-newrelic/assets/670701/0def02f3-ebde-42fd-a1a9-06ba2159631a">

There are **no documentation** on how to import `newrelic_notification_channel` and if it's possible or why it's not possible.

### Expected Behavior

Documented import

### Steps to Reproduce
N/A

### Debug Output
N/A

### Panic Output
N/A

### Important Factoids
N/A

### References
https://registry.terraform.io/providers/newrelic/newrelic/latest/docs/resources/notification_destination#additional-information
https://registry.terraform.io/providers/newrelic/newrelic/latest/docs/resources/notification_channel#additional-information

**Comment by amaor-newrelic** [comment]
[This comment](https://github.com/newrelic/terraform-provider-newrelic/issues/2035#issuecomment-1323751129) should help you get destination id and everything involved with slack.
As for channels, they are only created in the UI as part of a workflow. So if you've already created the channel, then you've already created the workflow too. And channels can only be used by a single workflow, so theres no real use in importing them. Additionally the channel id isn't available via the UI, and you'd need to look it up with the `channels` query in the NerdGraph API. 
All around quite cumbersome, so I'm not sure what the usecase is for importing a channel. 
It is possible, though, and we'll reconsider documenting everything I've mentioned here :D

**Comment by amaor-newrelic** [comment]
More updated destination id image -
<img width="1328" alt="image" src="https://github.com/newrelic/terraform-provider-newrelic/assets/89348176/4aabeba0-5024-47bb-a024-12f3651ca4ab">

**Comment by Maxwell2022** [comment]
thanks, I found it after a while. This issue is to fix the documentation adding this missing section with explanations.

**Comment by amaor-newrelic** [comment]
Fix for documentation is on the way, thank you for bringing this to our attention!

**Comment by pranav-new-relic** [comment]
Changes with an `Import` section in the documentation of the resources [`newrelic_notification_channel`](https://registry.terraform.io/providers/newrelic/newrelic/3.32.0/docs/resources/notification_channel#import) and [`newrelic_notification_destination`](https://registry.terraform.io/providers/newrelic/newrelic/3.32.0/docs/resources/notification_destination#import) have been released with v3.32.0 of the Terraform Provider. Thank you for reporting this issue :)
